### ISSUE TITLE
Downloads fail [ManhwaHentai]

### ISSUE THREAD

**Issue by bloodsign** [fmd-project-team/FMD#119]
Same issue with [RawQQ][https://github.com/fmd-project-team/FMD/issues/87](url)
FDM can retrieve the list of chapters from manhwahentai.com but the download fails. 
Here's the manhwa i'm trying to download: https://manhwahentai.com/manhwa/liliths-cord/
I've tried the CF workaround but it didn't work for me, possibly a site change/need to update lua script?

**Comment by bloodsign** [comment]
Also, I can't seem to figure out which lua script manhwahentai uses from the modules folder.

**Comment by bloodsign** [comment]
Okay, I tried to retry the failed downloads again and it seems to be working now. Beware about downloading multiple series from ManhwaHentai all at the same time, it might cause the download to fail again. (I tried to redownload 3 series at the same time, the 3rd series failed, but when the 1st and 2nd series I redownloaded finished, the 3rd series started downloading successfully)

I will be temporarily be closing this issue since the problem seems to have fixed it self.
